source/changelog.txt

```diff
@@ -12,6 +12,43 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v1.8.0
+------
+
+*Released February 28, 2023*
+
+Autocomplete suggests completions for database level aggregation stages.
+
+Compatibility Changes
+~~~~~~~~~~~~~~~~~~~~~
+
+- ``EJSON.stringify`` no longer accepts a ``{{strict}}`` option.
+- These methods are removed:
+
+  - ObjectId.prototype.generate
+  - ObjectId.prototype.getInc
+  - ObjectId.prototype.get_inc
+  - ObjectId.getInc
+
+- Code objects store a string in their ``.code`` property. Code objects
+  do not store JavaScript functions in their ``.code`` property.
+- If an object passes a key to a database function, ``mongosh`` only
+  sends the object's own keys to the server. ``mongosh`` does not send
+  inherited enumerable keys to the server.
+
+Issues fixed
+~~~~~~~~~~~~
+
+- :issue:`MONGOSH-1358` Updates for the 5.1.0 :driver:`Node.js driver
+  </node/>`. 
+- :issue:`MONGOSH-1336` Performance improvement some use cases including
+  the ``--version`` and ``--build-info`` flags.
+- :issue:`MONGOSH-1316` Surfaces the ``createEncryptedCollection``
```

Comment:
I am assuming that an `issue` means a bug, following which this one is not really an `issue` but more like a small feature. Please let me know if I am interpreting it correctly.

Reply:
Not really, that's the heading we put at the top of the list on [the server pages](https://www.mongodb.com/docs/manual/release-notes/6.0/). Those pages also have section headers (like "Compatibility Changes" here.) 

mongosh releases don't follow the same process though. I've changed the header to make it more general.

Reply:
Thanks @davemungo